vizro-core/examples/_chart-gallery/pages/_factories.py

```diff
@@ -74,3 +74,39 @@ def butterfly_factory(group: str):
             make_code_clipboard_from_py_file("butterfly.py"),
         ],
     )
+
+
+def connected_scatter_factory(group: str):
+    """Reusable function to create the page content for the column chart with a unique ID."""
+    return vm.Page(
+        id=f"{group}-connected-scatter",
+        path=f"{group}/connected-scatter",
+        title="Connected Scatter",
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
+                text="""
+                #### What is a connected scatter chart?
+
+                A connected scatter chart visualizes two variables using dots, with lines connecting the dots in the
+                order of the data points. One variable is plotted along the x-axis and the other along the y-axis,
+                showing both the relationship and the sequence of the data.
+
+                &nbsp;
```

Comment:
```
                &nbsp;
```
Does this render OK? Do you need it, and why?

Reply:
You need it for an extra empty line :) This is how you do it inside the markdown text 👍

Reply:
Ah, perfect. I always use <br /> but maybe I shouldn't!